src/main/java/unicash/ui/SummaryWindow.java

```diff
@@ -69,28 +81,84 @@ public SummaryWindow() {
      *     </ul>
      */
     public void show(HashMap<String, Double> expenseSummary) {
-        Stage root = getRoot();
         setPieChart(expenseSummary);
```

Comment:
Should we be setting the pie chart if we haven't confirmed that expense summary is not empty? Maybe we can move this down into the if statement

Reply:
Actually, I'm wondering why we set the pie chart here? Since `CommandResult` already does the same thing. Seems like the call path might be a bit misordered, good to simplify this

Reply:
You're right. There isn't a need to call that method here. The method was already called before by `MainWindow#executeCommand`. I have removed this method call here, thanks for pointing this out!